1001-2000/LIT1558Matthew.xml

```diff
@@ -68,26 +71,63 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      <ptr target="bm:Nicoll1821Bodleianae"/>
                      <citedRange unit="page">17</citedRange>
                   </bibl>
+                  <bibl>
+                     <ptr target="bm:Zuurmond1989EvangeliumMarc"/>
+                     <citedRange unit="page">22-23, 26-27, 29</citedRange>
+                  </bibl>
                </listBibl>
-
-               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="BiographyMatthew">
-                  <label>Biography of Matthew</label>
-               </div>
+              <note>The incipit and explicit are taken from
+                 <ref type="mss" corresp="GCA001"/>
+               </note>
             </div>
 
-            <!--Is the Note on the division of the Gospel of Matthew part of the Biography or the Tituli or is it an own textpart?-->
-
-            <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="TituliMatthew">
-               <label>Tituli of the Gospel of Matthew</label>
+             <div type="textpart" subtype="textsubunit" xml:lang="gez" xml:id="BiographyMatthew">
+                <label>Biography of Matthew</label>
+                <div type="textpart" subtype="incipit"><ab>ወለማቴዎስ፡ ስሙ፡ ሌዊ፡ ወትርጓሜሁ፡ ኅሩይ፡ ወውእቱ፡
+                 እምነገደ፡ ይሳኮር፡ እምሀገረ፡ ናዝሬት። ስመ፡ አቡሁ፡ ዲቁ፡ ወስመ፡ እሙ፡ ካሩትያስ።</ab></div>
+                <div type="textpart" subtype="explicit"><ab>ወሰበከ፡ ባቲ፡ በህንድ፡ ወበኢየሩሳሌም።</ab></div>
+             </div>
+             <div type="textpart" subtype="textsubunit" xml:lang="gez" xml:id="EusebianSectionsMatthew">
+                <label>Statement about Eusebian Sections in the Gospel of Matthew</label>
+                <div type="textpart" subtype="incipit"><ab>ወኍልቈ፡ ምዕራፋቲሁ፡ ንኡሳን፡ ፫፻፶፭፡</ab></div>
+                <div type="textpart" subtype="explicit"><ab>ወምዕራፉ፡ በልሳነ፡ ሮሜ። ፷፰፡</ab></div>
+             </div>
+             <div type="textpart" subtype="textsubunit" xml:lang="gez" xml:id="TituliMatthew">
+                <label>Tituli of the Gospel of Matthew</label>
+                <div type="textpart" subtype="incipit"><ab>
+                <l n="1">፩፡ ሰብአ፡ ሰገል።</l>
+                <l n="2">፪፡ ቅትለተ፡ ሕፃናት።</l>
+                <l n="3">፫፡ ስብከተ፡ ዮሓንስ።</l></ab></div>
+                <div type="textpart" subtype="explicit"><ab>
+                <l n="66">፷፮፡ ክሕደተ፡ ጴጥሮስ።</l>
+                <l n="67">፷፯፡ ነስሖተ፡ ይሁዳ።</l>
+                <l n="68">፷፰፡ ዘከመ፡ ነሥአ፡ ዮሴፍ፡ራማዊ፡ ሥጋሁ፡ ለእግዚእ።</l></ab></div>
+              </div>
+              <div type="textpart" subtype="chapter" xml:lang="gez" corresp="LIT2709Matthew">
+                <label>Bǝsrāta Mātewos</label>
+               <div type="textpart" subtype="subtextunit" xml:id="MatthewGospelHeadline">
+                 <label>Headline of the Gospel of Matthew</label>
+                  <ab>በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩፡ አምላክ። <seg type="title inscriptio expanded">ብስራተ፡ ወንጌል፡
+                   ማቴዎስ፡ ሐዋርያ። አሓዱ፡ እምዓሠርቱ፡ ወክልኤቱ።</seg> ጸሎቱ፡ ወበረከቱ፡ የሀሉ፡ ምስለ፡ ገብሩ፡ በእደ፡ ማርያም፡ ዘመለኮት።
```

Comment:
Here also (as Massimo stated below), you could delete the proper name and state it only in the manuscript record.
In the work record you could encode: ምስለ፡ ገብሩ፡ `<gap reason="ellipsis" resp="Abr"/>`

In the manuscript record you could also encode the fact that the third part of the name was added interlineally (by a different hand - I don't know how this fact should be encoded? I was not able to find it in the guidelines): የሀሉ፡ ምስለ፡ ገብሩ፡ በእደ፡ ማርያም፡ `<add place="interlinear">ዘመለኮት።</add>`




<gap reason="ellipsis" resp="Abr"/>

Reply:
I already encoded in the manuscript description that the name Zamalakot was written interlineary. But I do think that this is a second hand. This is confirmed by the subscriptions (and the colophons) in the rest of the manuscript, where the two persons are associated and their names are written with the same hand

Reply:
You are right, but still the writing and ink of this addition looks different to me. However, it doesn't matter as this information doesn't change anything.

Reply:
It could be, Sanne. But this does not add anything to our description, because Zamalakot was clearly involved in the original project of the manuscript, that is since the beginning of its production.